src/components/voting/votingBar.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import grid from 'flexboxgrid/dist/flexboxgrid.css';
+
+import votingConst from '../../constants/voting';
+import style from './votingBar.css';
+
+const VotingBar = ({ votes }) => {
+  const { maxCountOfVotes, maxCountOfVotesInOneTurn } = votingConst;
```

Comment:
To prevent verbosity please perform this shorthand assignment while importing

Reply:
I wanted to use it like that, but it doesn't work. 
```
import { maxCountOfVotes, maxCountOfVotesInOneTurn } from '../../constants/voting';
```
ends up in both `maxCountOfVotes` and `maxCountOfVotesInOneTurn` being `undefined`